libethcore/KeyManager.cpp

```diff
@@ -342,6 +342,19 @@ string const& KeyManager::accountName(Address const& _address) const
 	}
 }
 
+void KeyManager::changeName(Address const& _address, std::string const& _name)
+{
+	try
+	{
+		m_keyInfo.at(_address).accountName = _name;
+	}
+	catch (...)
+	{
+		return;
+	}
+	write(m_keysFile);
```

Comment:
CommonIO.
I'm saying that, this method behaves exactly like any other method in this class. If write fails, it will fail in all places where methods using `write` are used. I don't want to make changing/handling this behaviour part of this pr.